Gems/ROS2/Code/Source/ContactSensor/ROS2ContactSensorComponent.h

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Should we provide documentation, in Doxygen format, for every member and method?